src/main/java/br/com/shapeup/adapters/output/repository/jpa/friend/FriendshipMongoRepository.java

```diff
@@ -20,4 +20,6 @@ public interface FriendshipMongoRepository extends MongoRepository<FriendshipReq
 
     @Query("{'$or':[ {'usernameSender':?0}, {'usernameReceiver':?0} ]}")
     List<FriendshipRequestDocument> findAllByUsernameSenderOrUsernameReceiverEqualsIgnoreCase(String username);
+
+    void deleteAllByUsernameSenderOrUsernameReceiverEqualsIgnoreCase(String usernameSender, String usernameReceiver);
```

Comment:
aqui eu tentei usar a query de cima do or, mas não funcionou, deu erro de "Failed to instantiate void using constructor NO_CONSTRUCTOR with arguments", não sei se mosquei forte pq são 5h da manhã, mas enfim, deixei anotado que vou melhorar esse código no futuro

Reply:
tranquilo, tambem caguei aqui numas coisas, devido a apresentacao, acho justo, mas depois podemos ver com calma e melhorar, falo isso mais pelo texto seu do que o codigo, ta?!